typings/missing-types.d.ts

```diff
@@ -1,2 +1,4 @@
 declare module 'synthetix-data';
+declare module '@synthetixio/providers';
```

Comment:
if this is needed then the `@synthetixio/providers` package should be looked at.